### ISSUE TITLE
Latest files?

### ISSUE THREAD

**Issue by xloem** [tseliot/nvidia-graphics-drivers#55]
I’m using version 470.182.03 from the mirrors but only seeing 470.161.03 .

As an aside, I’ve been looking a little bit at patching for kernel 6.3 (so far found that vma_flags use setter functions now, and dumb_destroy for drm gem allocations is removed), is there existing work for this anywhere, or is my work useful as a partial contribution somewhere?

**Comment by tseliot** [comment]
Yes, sorry, I was convinced I had already pushed the changes. It should all be up-to-date now.
